alerting/src/test/kotlin/org/opensearch/alerting/resthandler/SecureMonitorRestApiIT.kt

```diff
@@ -430,6 +506,34 @@ class SecureMonitorRestApiIT : AlertingRestTestCase() {
         }
     }
 
+    fun `test ack alerts with an user with ack alerts role`() {
```

Comment:
This test is only checking if the user with `ALERTING_ACK_ALERTS` role can view the alerts and not acknowledging the alerts. Please check for that.

Reply:
Will add as a part of another test in the next PR?

Reply:
Renamed the test as `test get alerts with an user with get alerts role` and changed the role to get alerts role.